test.d

```diff
@@ -187,24 +184,26 @@ private:
 
     bool libclangExists (const ref Clang clang)
     {
-        auto libclangPath = Path.join(workingDirectory, clang.versionedLibclang);
-        return Path.exists(libclangPath);
+        auto libclangPath = buildNormalizedPath(workingDirectory, clang.versionedLibclang);
+        return exists(libclangPath);
     }
 
     void download (const ref Clang clang)
     {
         auto url = clang.baseUrl ~ clang.filename;
         auto dest = archivePath(clang.filename);
 
-        if (!Path.exists(dest))
-            Http.download(url, dest);
+        import std.net.curl;
```

Comment:
> Would you be ok with using vibe.http.client

Wasn't the point of this PR to minimze the dependencies 😉.

> Alternatively, I could use Adam's simple http module which depends only on std.socket

Sounds better. Do you have a link?